package.json

```diff
@@ -72,6 +72,7 @@
     "fmt": "yarn run eslint --fix && yarn run prettier --write",
     "lint": "yarn run eslint && yarn run prettier --check",
     "prettier": "prettier './src/**/*.{ts,tsx}'",
+    "proxy:run": "./src-tauri/bin/grpcproxy-aarch64-apple-darwin",
```

Comment:
how we can generalize for all arch?

Reply:
`make run` is the answer. It requires golang installed on the machine though.

Reply:
done